packages/domain-search/src/components/domains-mini-cart/actions.tsx

```diff
@@ -0,0 +1,24 @@
+import { __experimentalHStack as HStack, Button } from '@wordpress/components';
+import { useI18n } from '@wordpress/react-i18n';
+import { useDomainSearch } from '../domain-search';
+
+export const DomainsMiniCartActions = () => {
+	const { onContinue, openFullCart } = useDomainSearch();
```

Comment:
Should we consider this kind of component, such as primitive components, without side effects?
What if we want to reuse it somewhere else (since we are exporting it), relying on structure/styling consistency, but with different onClick handlers.

Maybe we should introduce onClick handlers as a property and control them from the parent components?
What do you think?